src/tests/ftest/soak/soak.yaml

```diff
@@ -1,39 +1,69 @@
 hosts:
-  test_servers:
-    - server-A
-    - server-B
-    - server-C
-    - server-D
-  client_partition: daos_client
-# This timeout must be longer than the test_timeout param (+30minutes)
-timeout: 5400
+# servers if no server partition is defined
+    test_servers:
+      - server-A
+      - server-B
+      - server-C
+      - server-D
+# servers if a server partition is defined
+    # server_partition: daos_server
+    client_partition: daos_client
+orterun:
+    allow_run_as_root: True
+# include Cluster specific slurm params
+srun:
+    reservation:
+srun_params:
+    reservation:
+# This timeout must be longer than the test_timeout param (+15minutes)
+# 2 hour test
+timeout: 8000
 logdir: /tmp/soak
 server_config:
+#    access_points: ['wolf-118:10001']
```

Comment:
is this left intentionally?

Reply:
I did leave it there intentionally as to the format of the access point; can I leave it or should it be removed?

Reply:
I removed the comments